focus-blocked/js/main.js

```diff
@@ -147,7 +147,13 @@ try {
     }
   });
 
-  if (cuddly_bear_mode && !strict_blocking) {
+  let focusRemainingSeconds = focus_mode_end_time.diff(moment(), 'seconds');
+  if (
+    (cuddly_bear_mode ||
+      shouldActivateAlwaysBlock ||
+      focusRemainingSeconds <= 0) &&
```

Comment:
If remaining seconds is less than zero, the block is over and it should show unblocked?